src/plone/restapi/tests/test_types.py

```diff
@@ -108,6 +108,20 @@ def test_get_jsonschema_for_fti(self):
         )
         self.assertNotIn('title', jsonschema['properties'].keys())
 
+    def test_get_jsonschema_for_fti_non_dx(self):
```

Comment:
@jaroel any reason why you didn't add a test here that just calls @types/Plone%20Site (I guess it kind of sucks that we use blanks in our type ids)

Reply:
No reason. Using the Plone Site FTI is a beter idea. I've changed the test to reflect that.